tests/storage/restore.go

```diff
@@ -301,14 +302,16 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 		})
 
 		Context("and no snapshot", func() {
-			It("[test_id:5255]should reject restore", func() {
+			It("should wait for snapshot to be exist and be ready", func() {
 				vm, err = virtClient.VirtualMachine(testsuite.GetTestNamespace(nil)).Create(context.Background(), vm, metav1.CreateOptions{})
 				Expect(err).ToNot(HaveOccurred())
-				restore := createRestoreDef(vm.Name, "foobar")
+				restore := createRestoreDef(vm.Name, fmt.Sprintf("snapshot-%s", vm.Name))
```

Comment:
restore called `snapshot-%s` a little weird no?

Reply:
just followed the same logic as in other tests..

Reply:
sorry I thought the restore is called snapshot-something but that's the `snapshotName`